e2e/test/scenarios/embedding/embedding-questions.cy.spec.js

```diff
@@ -198,24 +198,42 @@ describe("scenarios > embedding > questions", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.contains("October 7, 2023, 1:34 AM");
   });
+});
+
+describeEE("scenarios [EE] > embedding > questions", () => {
```

Comment:
The previous test was in `describe`, so I need to move it inside `describeEE`. This uses the same setup as `describe` but add `setTokenFeatures("all")`